README.md

```diff
@@ -0,0 +1,72 @@
+# Cloudflare Worker Test Harness
+
+### Usage
+
+- Add it to your `package.json` file.
+
+  ```js
+  {
+    // ...
+
+    "devDependencies": {
+      "cloudflare-worker-test-harness": "git://github.com/envato/cloudflare-worker-test-harness.git#master"
```

Comment:
why not publish it to npm? If it's not stable yet but maybe can publish beta versions?

Reply:
Apologies, I just hit save on my description here 😛 Once we get it onto `master` I will publish it. This PR is just getting it in and everyone happy and then I'll update our docs with the publish.